extension/commands/openTransactionViewCommand.ts

```diff
@@ -27,7 +27,9 @@ import { GlobalState } from '../util/GlobalState';
 export async function openTransactionView(treeItem?: InstantiatedTreeItem): Promise<void> {
     const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
     outputAdapter.log(LogType.INFO, undefined, `Open Transaction View`);
-    let smartContract: string;
+    let smartContractLabel: string;
+    let contract: { name: string, contractInstance: {}, transactions: Array<any>, info: {} };
+    let data: { name: string, version: string, channel: string, label: string, transactions: Array<any>, namespace: string };
```

Comment:
If this is the same as `ISmartContract`, could we create the interface in the extension and then copy the interface file into the `src` directory on build? (Like we do with ExtensionCommands.ts in the build npm script)